plugin/trino-exchange-filesystem/pom.xml

```diff
@@ -355,4 +355,28 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <profiles>
```

Comment:
`CephClient` is currently in `trino-testing-containers` because tests for other connectors might use it. Also, if we want to add product tests (I would like to add product tests in separate PR), it makes sense to put `CephClient` in `trino-testing-containers`.

This change resolves an issue with the `duplicate-finder-maven-plugin`.